### ISSUE TITLE
Merge `helm template` plugin into core Helm

### ISSUE THREAD

**Issue by kfox1111** [kubernetes/helm#1887]
I'm trying to add helm package rendering validation to some of our lightweight gate jobs. Finding the following issues:
 * helm install xxx --dry-run --debug produces the yaml to be loaded, but also some non yaml at the beginning that is nontrivial to parse out. There maybe needs to be another flag to say, "only give me the yaml that will be uploaded"
 * helm install xxx --dry-run --debug requires a working k8s/tiller. This is a heavy lift for a lightweight validation job
 * helm template plugin. currently requires to be built from source. this is painful from a lightweight validation job
 * helm template plugin also runs the risk of being out of sync with the helm intended to be used.

Ideally, I think as an operator, I'd like to see the helm template plugin merged into the main helm client binary. Being able to see what its going to do is a very important piece of debugging functionality for operators. Would that be possible? I don't think it would add much code to the helm cli?

**Comment by technosophos** [comment]
I could see doing this, since it actually is only a tiny bit of new code. I believe this would also be fairly easy to do, so I am marking this as a starter.

**Comment by chancez** [comment]
@technosophos I would love to see this. I'm basically begging someone to do this in slack 💯

**Comment by bonhamjp** [comment]
This issue seems to be a duplicate of #2755 which has an open PR. Should this issue be closed? 

I did start working on this, not knowing it was a duplicate. I'm thinking I should have asked before starting in on it, but since it is still open someone else might get confused.

**Comment by bacongobbler** [comment]
Yes, let's close this duplicate ticket.
